### PR TITLE
[Issue-3170] Add script for issue 3170 [internal review]

### PR DESCRIPTION
**Pull request by VjKlepikov**
ATF Test Scripts to check #[issue number in sdl_core repository]

This PR is **[ready]** for review.

### Summary
New ATF scripts to check the issue.

### ATF version
develop

### Changelog
Added new scripts
Updated test set for the issues in scope of 7.0 release

### CLA
- [X] I have signed [the CLA](https://docs.google.com/forms/d/e/1FAIpQLSdsgJY33VByaX482zHzi-xUm49JNnmuJOyAM6uegPQ2LXYVfA/viewform)

### CODE REVIEW

File: test_scripts/Defects/7_0/3170_AddCommand_DeleteCommand_multiple_times.lua [hunk 1]
```diff
@@ -0,0 +1,100 @@
+---------------------------------------------------------------------------------------------------
+-- Issue: https://github.com/smartdevicelink/sdl_core/issues/3170
+---------------------------------------------------------------------------------------------------
+-- Description: Processing of 'AddCommand' and 'DeleteCommand' RPCs multiple times
+--
+-- Preconditions:
+-- 1. Clean environment
+-- 2. SDL, HMI, Mobile session are started
+-- 4. App is registered
```
Comment (dboltovskyi) [inline test_scripts/Defects/7_0/3170_AddCommand_DeleteCommand_multiple_times.lua:9]: Please correct numbering

Comment (dboltovskyi) [inline test_scripts/Defects/7_0/3170_AddCommand_DeleteCommand_multiple_times.lua:9]: Please correct numbering

File: test_scripts/Defects/7_0/3170_AddCommand_DeleteCommand_multiple_times.lua [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+-- 2. HMI sends UI and VR part of response with "SUCCESS" result code
+-- SDL does:
+-- send AddCommand response with (resultCode: SUCCESS, success:true) to mobile App
+-- 3. Send DeleteCommand mobile RPC with "vrCommands" and "menuParams" data from App
+-- SDL does:
+-- transfer the UI part of request to HMI
+-- transfer the VR part of request to HMI
+-- 4. HMI sends UI and VR part of response with "SUCCESS" result code
+-- SDL does:
+-- send DeleteCommand response with (resultCode: SUCCESS, success:true) to mobile App
```
Comment (dboltovskyi) [inline test_scripts/Defects/7_0/3170_AddCommand_DeleteCommand_multiple_times.lua:25]: Please add list dashes. E.g.:
```
SDL does:
  - expectation 1
  - expectation 2
```

**Review by dboltovskyi — APPROVED** [review]

File: test_scripts/Defects/7_0/3170_AddCommand_DeleteCommand_multiple_times.lua (hunk 1 above)
Comment (VjKlepikov) [reply test_scripts/Defects/7_0/3170_AddCommand_DeleteCommand_multiple_times.lua]: done

File: test_scripts/Defects/7_0/3170_AddCommand_DeleteCommand_multiple_times.lua (hunk 2 above)
Comment (VjKlepikov) [reply test_scripts/Defects/7_0/3170_AddCommand_DeleteCommand_multiple_times.lua]: done

**Pull request merged**